pyalgotrading/algobulls/api.py

```diff
@@ -335,7 +335,7 @@ def start_strategy_algotrading(self, strategy_code: str, start_timestamp: dt, en
                 TradingType.BACKTESTING: 'backDataDate'
             }
             execute_config = {
-                map_trading_type_to_date_key[trading_type]: [start_timestamp.astimezone(timezone.utc).isoformat(), end_timestamp.astimezone(timezone.utc).isoformat()],
+                map_trading_type_to_date_key[trading_type]: [start_timestamp.astimezone(timezone.utc).strftime("%Y-%m-%dT%H:%M:%S.000Z"), end_timestamp.astimezone(timezone.utc).strftime("%Y-%m-%dT%H:%M:%S.000Z")],
```

Comment:
Try this for the conversion - 
start_timestamp.astimezone(timezone.utc).replace(tzinfo=None).isoformat()

Reply:
Done